src/pages/constants.js

```diff
@@ -0,0 +1,10 @@
+export const coffeePrices = {
+  espresso: 10,
+  cappuccino: 19,
+  americano: 7
+};
+
+export const mocha = {
+  mochadiscount: '$4.00'
+};
+
```

Comment:
```suggestion
export const coffeePrices = {
  espresso: 10,
  cappuccino: 19,
  americano: 7,
  mochaDiscount: '$4.00',
};
```
you could add mocha to the same coffee object